app/src/main/java/ch/epfl/skysync/viewmodel/CalendarViewModel.kt

```diff
@@ -79,12 +81,22 @@ class CalendarViewModel(
    */
   private suspend fun refreshUserAndCalendars() {
     _user.value = userTable.get(uid, this::onError)!!
-    val currentAvCal = AvailabilityCalendar(userTable.retrieveAvailabilities(uid, this::onError))
+    var currentAvCal = AvailabilityCalendar(userTable.retrieveAvailabilities(uid, this::onError))
+    val flights = userTable.retrieveAssignedFlights(flightTable, uid, this::onError)
+    flights.forEach { flight ->
+      if (flight !is FinishedFlight) {
```

Comment:
status should also be set to `assigned` for any flight when affected not just for a `FinishedFlight`

Reply:
It's set to assigned for every flight that isn't finished